webcompat/form.py

```diff
@@ -164,6 +164,17 @@ def normalize_url(url):
 
 def normalize_metadata(metadata_value):
     """Normalize the metadata received from the form."""
+    # If it's a list, we are dealing with @extra_labels,
+    # so convert into a comma separated string.
```

Comment:
Not sure, I like this part. It seems a lot of juggling because of the type.

Reply:
What about leaving it as it was before and see the modification in `if extra_labels` https://github.com/webcompat/webcompat.com/pull/2007/files#diff-f1975977b06efb529b8c60f23a7bc4eeR252